doc/api/test.md

```diff
@@ -219,6 +219,49 @@ test('a test that creates asynchronous activity', (t) => {
 });
 ```
 
+## Running tests from the command line
+
+The Node.js test runner can be invoked from the command line by passing the
+[`--test`][] flag. By default, Node.js will recursively search the current
+directory for test files:
+
+```bash
+node --test
+```
+
+Alternatively, one or more paths can be provided as the final argument(s) to
+the Node.js command, as shown below.
+
+```bash
+node --test test1.js test2.mjs test/
+```
+
+In this example, the test runner will execute the files `test1.js` and
+`test2.mjs`. The test runner will also recursively search the `test/` directory
+for test files to execute.
+
+When searching for test files to execute, the test runner behaves as follows:
+
+* Any files explicitly provided by the user are executed.
+* `node_modules` directories are skipped unless explicitly provided by the
+  user.
+* `.js`, `.cjs`, and `.mjs` filenames matching the following patterns are
+  treated as test files:
+  * `test` - Files whose basename is the string `'test'`. Examples: `test.js`,
+    `test.cjs`, `test.mjs`.
+  * `test-.+` - Files whose basename starts with the string `'test-'` followed
+    by one or more characters. Examples: `test-example.js`,
+    `test-another-example.mjs`.
+  * `.+[\.\-\_]test` - Files whose basename ends with one or more characters
```

Comment:
If we're using the regex notation, we should specify it's validating the end of the string (or the whole string)
```suggestion
  * `.[\.\-\_]test$` - Files whose basename ends with one character
```
```suggestion
  * `^.+[\.\-\_]test$` - Files whose basename consist of one or more characters
```